docker-compose-cli.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
```

Comment:
This is used to initialise the CLI with a containerised redis instance, but could include more stuff in the future (e.g. mocked object storage)